specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2018-08-01-preview/costmanagement.json

```diff
@@ -1947,6 +1947,234 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.CostManagement/alerts/updateStatus": {
+      "put": {
+        "tags": [
+          "Alerts"
+        ],
+        "operationId": "Alerts_UpdateStatusList",
+        "description": "Update alerts status for a subscription, limit to 100 alerts.",
+        "x-ms-examples": {
+          "Alert_List": {
+            "$ref": "./examples/AlertList.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/AlertListResult"
```

Comment:
Can you provide an example (captured from the service) in the comment that shows how the payload for this PUT will look like?

Reply:
sure, I'm working on it